static/bidder-params/adocean.json

```diff
@@ -6,7 +6,7 @@
   "properties": {
     "emiter": {
       "type": "string",
-      "description": "(in removal, use emitterPrefix instead) AdOcean emiter",
+      "description": "(Deprecated, use emitterPrefix instead) AdOcean emiter",
```

Comment:
@mmuras avoid parenthesis around Deprecated. AFAIK,  Deprecated will indicate bidder docs UI to treat this message in different way. So requesting to remove parenthesis and change description as mentioned below.


```
 "description": "Deprecated, use se emitterPrefix instead. AdOcean emiter"
```